contrib/snap/snapcraft.yaml

```diff
@@ -0,0 +1,66 @@
+name: dogecoin-core # you probably want to 'snapcraft register <name>'
+version: '1.14.3'
+summary: Reference client of Dogecoin, a peer-to-peer digital currency like Bitcoin.
+description: |
+  Dogecoin is a cryptocurrency like Bitcoin, although it does not use SHA256 as its proof of work (POW). Taking development cues from Tenebrix and Litecoin, Dogecoin currently employs a simplified variant of scrypt. MIT licenced.
+
+base: core18
+grade: stable
```

Comment:
Not sure about "stable" for development branch either?

Reply:
As mentioned in the above comment, this only works for reasons that have a formal github release. There are [four grades](https://snapcraft.io/docs/channels), and I thought stable fit the best.